codegen/templates/resource.go.tpl

```diff
@@ -198,6 +198,13 @@ func delete{{ .UpperCaseDifferentiator }}Resource(d *schema.ResourceData, meta i
 	log.Printf("[DEBUG] Deleted template auth backend role %q", vaultPath)
 	return nil
 }
+{{ else if .SupportsWrite }}
+func delete{{ .UpperCaseDifferentiator }}Resource(_ *schema.ResourceData, _ interface{}) error {
```

Comment:
I added this logic because with this resource was the first time I encountered an endpoint that supports Create and Update without supporting Delete.